suites/pacific/rados/tier-3_rados_test-location-stretch-mode.yaml

```diff
@@ -267,3 +267,25 @@ tests:
         tiebreaker_mon_site_name: arbiter
         delete_pool: true
       desc: Test the cluster when the arbiter site is rebooted
+
+  - test:
+      name: Mon replacement on Data site
+      module: test_stretch_mon_replacements.py
+      polarion-id: CEPH-83574971
+      config:
+        pool_name: test_stretch_pool5
+        replacement_site: DC1
+        tiebreaker_mon_site_name: arbiter
+        delete_pool: true
+      desc: Test stretch Cluster mon replacement - Data site
+
+  - test:
+      name: Mon replacement on Arbiter site
+      module: test_stretch_mon_replacements.py
+      polarion-id: CEPH-83574971
```

Comment:
Are both the test cases part of the same polarion ID?

Reply:
Yes. The test case speaks about mon replacement. Internally we have different procedures for data site mon and Arbiter mn, since it's tiebreaker.